app/views/host_aggregate/show.html.haml

```diff
@@ -0,0 +1,18 @@
+#main_div
+  - if @showtype == "performance"
+    = render :partial => "layouts/performance"
+    :javascript
+      var miq_after_onload = "miqAsyncAjax('#{ url_for(:action => @ajax_action, :id => @record.id)}');"
+  - elsif @showtype == "download_pdf"
+    = render :partial => "layouts/show_pdf"
+  - else
+    - if %w(instances hosts).include?(@display) && @showtype != "compare"
+      = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@host_aggregate.id}"}
+    - elsif @showtype == "compare" || @showtype == "drift_history" || @showtype == "drift" || @showtype == "item"
+      = render :partial => "layouts/#{@showtype}"
+    - elsif @showtype == "timeline"
+      = render :partial => "layouts/tl_show"
+      :javascript
+        var miq_after_onload = "miqAsyncAjax('#{ url_for(:action => @ajax_action, :id => @record.id)}');"
```

Comment:
OK. Makes sense. This was pulled from the AZ UI, with stuff not relevant for aggregates removed. I think these items were in the AZ partial that shouldn't have been there either. I'll remove the reference to drift, drift_history, and compare -- I think the rest of what's here is still needed.